rules/linux/process_creation/proc_creation_lnx_esxcli_firewall_disable.yml

```
@@ -0,0 +1,29 @@
+title: ESXi Firewall Disabled via ESXCLI
+id: 18fba7a0-8f63-49d3-9fc4-6192fe34793c
+status: experimental
+description: Detects when the ESXi firewall is disabled via esxcli.
+references:
+    - https://developer.broadcom.com/xapis/esxcli-command-reference/7.0.0/namespace/esxcli_network.html
+    - https://lolesxi-project.github.io/LOLESXi/lolesxi/Binaries/esxcli/
+author: Nathan Burns
+date: 2024-11-20
+tags:
+    - attack.t1562.004
+logsource:
+    category: process_creation
+    product: linux
+detection:
+    selection_img:
+        Image|endswith: '/esxcli'
+        CommandLine|contains|all:
+            - 'network'
+            - 'firewall'
+            - 'set'
+    selection_enable_switch:
+        CommandLine|contains:
+            - '-e false'
+            - '--enabled false'
```

```suggestion
            - 'set'
            - 'false'
    selection_enable_switch:
        CommandLine|contains:
            - '--enabled'
            - '-e'
```
To keep the logic with  e0f2e697-0352-49a3-b488-11b3dcf1c9fd